waves-scripts

```diff
@@ -82,9 +82,10 @@ start_bobtimus () {
 
     set +e
     find ./waves/dist ./waves/dist/index.html &> /dev/null
+    [ $? -ne 0 ] && is_built=false
     set -e
 
-    if [ $? -ne 0 ]; then
+    if ! $is_built; then
```

Comment:
I think we should not be implicitly building the frontend here because we are not serving it through bobtimus in development anyway.